docs/upgrading.rst

```diff
@@ -41,7 +41,6 @@ Use the following patterns to identify the deprecated environment variables in a
 
     git grep -e "DATADOG_[A-Z_]*" \
       -e "DATADOG_SERVICE_NAME" \
-      -e "DD_CALL_BASIC_CONFIG" \
```

Comment:
These upgrade instructions are specific to 0.x, so we might want to leave them unchanged. But for 2.x, we should make sure to have upgrade instructions for 1.x users.

Reply:
Good catch - I'll revert this change and make a separate PR for upgrade docs.